pallets/merkle-airdrop/src/lib.rs

```diff
@@ -402,47 +441,61 @@ pub mod pallet {
             origin: OriginFor<T>,
```

Comment:
I think the documentation should be adjusted to the vesting functionality now. And maybe the documentation of the module should mention this too :)